apps/meteor/ee/app/livechat-enterprise/server/lib/LivechatEnterprise.js

```diff
@@ -228,6 +228,12 @@ export const LivechatEnterprise = {
 				});
 			}
 		}
+		console.log(department?.archived, departmentData.enabled);
+		if (department?.archived && departmentData.enabled) {
```

Comment:
Is enabling the only action that we'll prevent on archived deps? Can we do other stuff like changing agents, names, settings, etc 👀 

Otherwise, would be good to just block everything apart from removing agents 🤔

Reply:
I've done it in this way since there are a couple of operations that should be available for archived departments (Like removing agents, removing from units/tags, etc...)